config/locales/server.en.yml

```diff
@@ -2253,6 +2253,7 @@ en:
       max_username_length_exists: "You cannot set the maximum username length below the longest username (%{username})."
       max_username_length_range: "You cannot set the maximum below the minimum."
       invalid_hex_value: "Color values have to be 6-digit hexadecimal codes."
+      empty_selectable_avatars: "You must first upload selectable avatars before enabling this setting."
```

Comment:
Let's enforce at least 2 avatars in the list otherwise there's no point in enabling this feature to select just 1 avatar.